native/xqlitenif/src/lib.rs

```diff
@@ -7,12 +7,14 @@ mod open;
 mod pragma_get;
 mod pragma_put;
 mod query;
+mod r2d2;
 mod shared;
 
+use crate::r2d2::XqliteConn;
 use crate::shared::XqliteConnection;
 
 fn on_load(env: Env, _info: Term) -> bool {
-    env.register::<XqliteConnection>().is_ok()
+    env.register::<XqliteConn>().is_ok() && env.register::<XqliteConnection>().is_ok()
```

Comment:
@filmor If am reading `resource_impl`'s docs well, it means I can remove `env.register::<XqliteConn>().is_ok()` here?

Reply:
Yes

Reply:
Done: https://github.com/dimitarvp/xqlite/pull/9/commits/2a6976cac0949998ab6e13ea834379b929661f6c